service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/IdMapper.java

```diff
@@ -0,0 +1,20 @@
+package uk.nhs.adaptors.gp2gp.ehr.mapper;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import lombok.AllArgsConstructor;
+import uk.nhs.adaptors.gp2gp.common.service.RandomIdGeneratorService;
+
+@AllArgsConstructor()
+public class IdMapper {
```

Comment:
Should we change as part of this story that existing mappers use `IdMapper` instead of `RandomIdGeneratorService`?

Reply:
Yes, NarrativeStatementMapper was updated to use IdMapper, Do we have more mappers?